src/pages/proposal.tsx

```diff
@@ -690,6 +690,8 @@ export const Proposal: React.FC = () => {
 
           <VotingTerminal
             status={proposal.status}
+            daoToken={daoToken}
+            blockNumber={proposal?.creationBlockNumber as number}
```

Comment:
The cast to `number` is probably redundant here
```suggestion
            blockNumber={proposal?.creationBlockNumber}
```